rules/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/quasilyte/go-ruleguard/rules
 go 1.15
 
 require (
-	github.com/quasilyte/go-ruleguard v0.3.1-0.20210203134552-1b5a410e1cc8
-	github.com/quasilyte/go-ruleguard/dsl v0.3.0
+	github.com/quasilyte/go-ruleguard v0.3.13
```

Comment:
@quasilyte , doing this would help to reduce the number of entries in golangci-lint:

https://github.com/golangci/golangci-lint/blob/master/go.sum#L621-L627

Reply:
I'm unclear why `go.sum` still has the `go-ruleguard/dsl v0.3.0` entry after the update:

```
github.com/quasilyte/go-ruleguard/dsl v0.3.0/go.mod h1:KeCP03KrjuSO0H1kTuZQCWlQPulDV6YMIXmpQss17rU=
```